.github/workflows/ci.yml

```diff
@@ -19,17 +19,20 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8
```

Comment:
@jelhan out of curiosity, why not using pnpm v9 right away?

Reply:
I followed the setup of the latest Ember CLI blueprints: https://github.com/ember-cli/ember-cli/blob/6691ea1008dd734a9c18c0ec5ed7a389987f608b/blueprints/addon/files/.github/workflows/ci.yml#L22-L24